pkg/kubelet/cm/devicemanager/checkpoint/checkpoint.go

```diff
@@ -96,10 +135,19 @@ func (cp *Data) UnmarshalCheckpoint(blob []byte) error {
 
 // VerifyChecksum verifies that passed checksum is same as calculated checksum
 func (cp *Data) VerifyChecksum() error {
+	if len(cp.Data.AllDevices) == 0 {
```

Comment:
Why doesn't use cp.Checksum.Verify?

Reply:
This is also to ensure that after the kubelet is upgraded, the check verify sum can be successful, and it will be treated as an object of the old version here.